src/packageUtils.ts

```diff
@@ -197,3 +206,9 @@ export function isSameDep(a: Dependency, b: Dependency) {
       a.depender.version === b.depender.version &&
       a.depender.packagePath === b.depender.packagePath;
 }
+
+export function isExternalPackage(pkg: string) {
+  return !['@atlaskit', '@atlassian', '@atlassiansox'].some(scope =>
```

Comment:
Instead of hardcoding like this, can we pass it in as a config? e.g.
```
ignoredPackagePatterns: [
    "@atlaskit/*",
    "@atlassian/*",
    "@atlassiansox/*"
]
```

Reply:
yep, can make it configurable